src/sentry/new_migrations/monkey/executor.py

```diff
@@ -37,7 +37,9 @@ def _check_db_routing(migration):
         def _check_operations(operations):
             failed_ops = []
             for operation in operations:
-                if isinstance(operation, (FieldOperation, ModelOperation, RenameContentType)):
+                if isinstance(
+                    operation, (FieldOperation, ModelOperation, RenameContentType, IndexOperation)
```

Comment:
I had to add the `IndexOperation` here because otherwise I got the missing "`hints={'tables':..}` argument" for `AddIndex`. It seemed to me that since the `AddIndex` operation is model specific that I could put this here cc @wedamija

Reply:
That looks good to me